### PR TITLE
pixman: CVE-2022-44638

### PR DESCRIPTION
**Pull request by ddstreet**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
CVE-2022-44638

###### Change Log  

CVE-2022-44638

###### Does this affect the toolchain?  

no

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-44638

###### Test Methodology

- Pipeline build id: 305245

### CODE REVIEW

File: SPECS/pixman/pixman.spec [hunk 1]
```diff
@@ -1,10 +1,11 @@
 Summary:	pixel manipulation library.
 Name:		pixman
 Version:    0.36.0
-Release:    2%{?dist}
+Release:    3%{?dist}
 License:	MIT
 URL:        https://cgit.freedesktop.org/pixman/
 Source0:    https://xorg.freedesktop.org/archive/individual/lib/%{name}-%{version}.tar.bz2
+Patch0:     CVE-2022-44638.patch
```
Comment (rlmenge) [inline SPECS/pixman/pixman.spec:8]: Nit: spacing

**Review by rlmenge — APPROVED** [review]

**Review by rlmenge — COMMENTED** [review]
Please add pixman to title

File: SPECS/pixman/pixman.spec (hunk 1 above)
Comment (ddstreet) [reply SPECS/pixman/pixman.spec:8]: spacing?

Comment (rlmenge) [reply SPECS/pixman/pixman.spec:8]: The Summary, Name, version, release and urls usually all line up nicely. Not a change you added but just looks weird as is. Feel free to ignore if needed.

**Pull request merged**